src/snek5000_cbox/cbox.usr.f

```diff
@@ -103,24 +103,33 @@ subroutine userchk
       endif
 
       ! history points
-      if (mod(ISTEP,nit_hist).eq.0) then
-         if (.not. ifpert) then
-             call hpts()
-         else
 
-             call opcopy(vtmp(1,1),vtmp(1,2),vtmp(1,ndim),vx,vy,vz)
-             n = NX1*NY1*NZ1*NELV
-             call copy(ttmp,T,n)
-             call opcopy(vx, vy, vz, vxp, vyp, vzp)
-             call copy(T,TP,n)
+      nxyz1 = nx1*ny1*nz1
+      nxyz2 = nx2*ny2*nz2
+      
+      if (lhis.gt.1) then		
```

Comment:
`.gt.1` or `.gt.0`?

Reply:
`lhis` is the maximum number of history points, always nb_hist_points + 1.